package.json

```diff
@@ -70,11 +70,11 @@
     "@formatjs/intl-locale": "^2.4.46",
     "@formatjs/intl-numberformat": "^7.4.2",
     "@formatjs/intl-pluralrules": "^4.3.2",
-    "@ledgerhq/devices": "6.24.1",
+    "@ledgerhq/devices": "6.27.1",
     "@ledgerhq/errors": "6.10.0",
-    "@ledgerhq/hw-transport": "6.24.1",
-    "@ledgerhq/hw-transport-http": "6.27.0",
-    "@ledgerhq/live-common": "22.0.2",
+    "@ledgerhq/hw-transport": "6.27.1",
+    "@ledgerhq/hw-transport-http": "6.27.1",
```

Comment:
Is it relevant to bump these here? Are you actually using the new versions? (excepted live-common)

Reply:
It's a dedup cause the build refused to work without it...